src/diffusers/models/modeling_utils.py

```diff
@@ -829,6 +808,9 @@ def _get_model_file(
             and version.parse(version.parse(__version__).base_version) >= version.parse("0.15.0")
         ):
             try:
+                if commit_hash is not None and revision is None:
+                    revision = commit_hash
```

Comment:
Also possible to do `revision=revision or commit_hash` below in the `hf_hub_download` directly